.circleci/config.yml

```diff
@@ -123,15 +123,21 @@ workflows:
       - test:
           filters:
             branches:
-              ignore: staging
+              ignore:
+                - staging
+                - release
       - acceptance:
           filters:
             branches:
-              ignore: staging
+              ignore:
+                - staging
+                - release
       - build:
           filters:
             branches:
-              ignore: staging
+              ignore:
+                - staging
+                - release
```

Comment:
So, build doesn't actually do anything. By ignoring it, any steps that depend on build won't run. That means `publish_release_assets` in this case won't be executed.

Reply:
Ah, got it.